src/commonControls/listControl/ListControl.ts

```diff
@@ -143,6 +143,14 @@ export interface ListControlProps extends ControlProps {
      */
     inputHandling?: ControlInputHandlingProps;
 
+    /**
+     * Function that maps the slot-value IDs to rendered value that
+     * will be presented to the user as a list.
+     *
+     * Default: returns the slotValue unchanged.
+     */
+    valueRenderer?: (slotValue: string, input: ControlInput) => string;
```

Comment:
param -> 'value'.  `ListControl.state.value` is the thing, and it is defined as being a slotValueId.

Reply:
Fixed